<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lists</class>
 <widget class="QWidget" name="Lists">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="append_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="append_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>61</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Posicion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="append_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtValor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>221</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Valor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="append">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QListView" name="list1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
